app/routes.js

```diff
@@ -14,7 +17,7 @@ const app = express.Router()
 
 app.use(artsyRelayMiddleware)
 
-app.get('/artist/:id', (req: $Request, res: $Response, NextFunction) => {
+app.get('/artist/:id', (req: $Request, res: $Response, next: NextFunction) => {
```

Comment:
if it doesn't we might want to set this on the project settings in VS Code

![screen shot 2016-12-29 at 2 17 30 pm](https://cloud.githubusercontent.com/assets/49038/21552716/90cdae02-cdd1-11e6-8ba3-fb7fee5459c6.png)